cmd/sippy-daemon/main.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/openshift/sippy/pkg/dataloader/prowloader/gcs"
+	"github.com/openshift/sippy/pkg/dataloader/prowloader/github"
+	"github.com/openshift/sippy/pkg/flags"
+	"github.com/openshift/sippy/pkg/github/commenter"
+	"github.com/openshift/sippy/pkg/sippyserver"
+)
+
+var logLevel = "info"
+
+type SippyDaemonFlags struct {
+	DBFlags          *flags.PostgresFlags
+	GoogleCloudFlags *flags.GoogleCloudFlags
+
+	GithubCommenterFlags *flags.GithubCommenterFlags
+	MetricsAddr          string
+}
+
+func NewSippyDaemonFlags() *SippyDaemonFlags {
```

Comment:
That's the only thing it does today but it looks like @neisw's intention was to have a generic server running long running tasks like the commenter, other stuff is easily added.

Reply:
Ah yes that rings a bell.

Reply:
Yes, I wrote it to be flexible.  I don't particularly mind if we give it a specific command for now, if that makes it easier to track / maintain, since that is the only thing we are using it for though.

Reply:
I'd rather just leave the name as is, this PR already touches a lot.  If we think it's worth renaming we can do it separately, is that OK?